content/guides/accessibility/color-considerations.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Color considerations
+description: Color is an important aspect of creating accessible, inclusive experiences. 
+---
+
+## Overview
+
+This guidance covers considerations that can't automatically be integrated into Primer. It supplements [accessibility guidance given in the Color portion of our Foundations documentation](https://primer.style/design/foundations/color#accessibility).
+
+### Design tokens
+
+The majority of color contrast considerations should be handled by [Primer color design tokens](https://primer.style/design/foundations/color#how-to-use-color-for-product-ui). Ensure that solutions for the following considerations integrate design tokens as a method of creating accessible experiences.
+
+### Determining contrast
+
+[WebAIM's Contrast Checker tool](https://webaim.org/resources/contrastchecker/) is the recommended way to calculate color contrast ratios.
+
+### The APCA
+
+The [Advanced Accessible Perceptual Contrast Algorithm](https://www.myndex.com/APCA/) (<abbr>APCA</abbr>) is a proposal and is **not normative**. In terms of WCAG, and therefore, MAS-C compliance, this means that color contrast is valid **only** if it meets WCAG 2.1 criteria. 
```

Comment:
WCAG 2.1 Level A and AA criteria"? Thinking of the number of questions we get around AAA/high contrast mode in Slack.

Reply:
Good call. Added this and language about what high contrast is in https://github.com/primer/design/pull/560/commits/dbc099e30d8702eb394d617af302260908983f58